packages/pyinstaller/PKGBUILD

```
@@ -2,41 +2,72 @@
 # See COPYING for license details.
 
 pkgname=pyinstaller
-_pkgname=PyInstaller
-pkgver=3.6
-pkgrel=4
+pkgver=6.11.1
+pkgrel=1
 epoch=2
 groups=('blackarch' 'blackarch-misc')
-pkgdesc='A program that converts (packages) Python programs into stand-alone executables, under Windows, Linux, Mac OS X, Solaris and AIX.'
-arch=('any')
+pkgdesc='Bundles a Python application and all its dependencies into a single package.'
+arch=('x86_64' 'aarch64')
 url='http://www.pyinstaller.org/downloads.html'
-license=('GPL')
-depends=('python2' 'python2-requests' 'python2-pyusb' 'python2-dis3'
-         'python2-pefile' 'python2-macholib' 'python2-setuptools'
-         'python2-altgraph')
-source=("https://files.pythonhosted.org/packages/source/${_pkgname::1}/$_pkgname/$_pkgname-$pkgver.tar.gz")
-sha512sums=('97e7ac3632b5a84a85268c68cb3f3a404ca9f2786396f31016313a15c81c9c453983fd9d0376de0e7543401d3ad8af58b839dde7cefb0dfe7cbb9f3ac360f3e0')
+license=('custom:mixed')
```

```suggestion
license=('Apache-2.0' 'GPL-2.0-or-later' 'MIT')
```
Ref.
https://github.com/pyinstaller/pyinstaller/blob/develop/COPYING.txt
https://pyinstaller.org/en/stable/license.html